Makefile

```diff
@@ -16,22 +16,41 @@ TARGET = cc-runtime
 DESTDIR :=
 PREFIX := /usr/local
 BINDIR := $(PREFIX)/bin
-QEMUBINDIR := $(BINDIR)
 SYSCONFDIR := $(PREFIX)/etc
 LIBEXECDIR := $(PREFIX)/libexec
 LOCALSTATEDIR := $(PREFIX)/var
 SHAREDIR := $(PREFIX)/share
 
 CCDIR := clear-containers
+
 PKGDATADIR := $(SHAREDIR)/$(CCDIR)
+PKGLIBDIR := $(LOCALSTATEDIR)/lib/$(CCDIR)
+PKGRUNDIR := $(LOCALSTATEDIR)/run/$(CCDIR)
 PKGLIBEXECDIR := $(LIBEXECDIR)/$(CCDIR)
 
+KERNELPATH := $(PKGDATADIR)/vmlinux.container
+IMAGEPATH := $(PKGDATADIR)/clear-containers.img
+
+QEMUBINDIR := $(BINDIR)
+QEMUCMD := qemu-lite-system-x86_64
```

Comment:
That's a good point, but these are just defaults: the package maintainer may need to set `QEMUCMD`. However, it's easy to detect CentOS/RHEL, so I'll add a test for this...